filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -330,8 +345,23 @@ func TestAuthorizeRequestFilter(t *testing.T) {
 
 						allow_body {
 							input.parsed_body.target_id == "123456"
-						}						
-					`,
+						}	
+						
+						decisionIdPresent {
```

Comment:
Nit: with rest of the naming followed in the policy this would be `decision_id_present`.  

Below would be another way to do it, if you like.
```
decision_id := input.attributes.metadataContext.filterMetadata.open_policy_agent.decision_id

allow_object_decision_id_in_header = response {
    input.parsed_path = ["allow", "structured"]
    decision_id 
    response := {
        "allowed": true,
        "response_headers_to_add": {
            "decision-id": decision_id
        }
    }
}
```